pom.xml

```diff
@@ -36,10 +36,19 @@
 		</dependency>
 
 		<dependency>
-			<groupId>org.springdoc</groupId>
```

Comment:
please don't delete, add or change the the pom.xml relative to the stage version
if something should be added to it, make a branch, apply the change there and make a pull request to stage